app/views/users/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<p id="notice"><%= notice %></p>
```

Comment:
it will be better to just give pop up message if the notice is only for deletion

Reply:
They won't go to show, if they are deleted.

Show is currently just a placeholder, nothing to show from a user yet I guess.